src/webportal/src/app/job-submission/components/tools/tensorboard.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {cloneDeep, isNil} from 'lodash';
+import {Hint} from '../sidebar/hint';
+import {ErrMsg} from '../sidebar/errormessage';
+import {generateDefaultTensorBoardExtras} from '../../utils/utils';
+import {TooltipIcon} from '../controls/tooltip-icon';
+import {
+  FontWeights,
+  Toggle,
+  Stack,
+  Text,
+  FontSizes,
+} from 'office-ui-fabric-react';
+
+const style = {
+  headerText: {
+    root: {
+      fontSize: FontSizes.large,
+      fontWeight: FontWeights.semibold,
+    },
+  },
+};
+
+export const TensorBoard = (props) => {
+  const {
+    jobData,
+    taskRoles,
+    extras,
+    onChange,
+  } = props;
+
+  const defaultLogPath = '/mnt/tensorboard';
```

Comment:
move this constant to utils/constants.js file

Reply:
Moved.